docker-compose.yml

```diff
@@ -77,6 +77,9 @@ services:
       - RACK_TIMEOUT_SERVICE_TIMEOUT=0
       - GITPOD_WORKSPACE_ID=${GITPOD_WORKSPACE_ID:-}
       - GITPOD_WORKSPACE_CLUSTER_HOST=${GITPOD_WORKSPACE_CLUSTER_HOST:-}
+      - ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY=bEZjYLY9tCYGh6WlcMtEJpIi7GO2plZC
+      - ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY=PpLWizzsQHIWnIihtECw8nDHZQd0amzf
+      - ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT=3S99KdpdEWLnCYudBgUfdCFDBWePWCud
```

Comment:
we probably also need these in the workers container now? if yes, it'd probably be nice to use yml anchors (?) for these values so we don't have to duplicate them